README.md

```diff
@@ -2,56 +2,66 @@
 
 The repo contains the API server and client for retrieving hardware information.
 
-
 ## Prerequisites for Deploying Locally
 
-* Install [docker](https://docs.docker.com/engine/install/ubuntu/) and [setup permissions](https://docs.docker.com/engine/install/linux-postinstall/) for server deployment
-* Install `rust` and `cargo` for client deployment (https://www.rust-lang.org/tools/install)
-
+- Install [docker](https://docs.docker.com/engine/install/ubuntu/) and
+  [setup permissions](https://docs.docker.com/engine/install/linux-postinstall/)
+  for server deployment <<<<<<< HEAD
+- Install `rust` and `cargo` for client deployment
+  (https://www.rust-lang.org/tools/install)
 
 ## Running API Server With Docker
 
-Go to the `server/` directory in the project. The following command builds the hwapi server:
+Go to the `server/` directory in the project. The following command builds the
+hwapi server:
 
 ```bash
-:server$ docker-compose up --attach-dependencies --build hwapi-dev
+docker-compose up --attach-dependencies --build hwapi-dev
```

Comment:
I’d like to keep the directory indicator to specify where to run this command

Reply:
But but then I can't copy and paste from the README?